core/src/dird/ua_audit.cc

```diff
@@ -142,4 +142,20 @@ void UaContext::LogAuditEventCmdline()
   Emsg3(M_AUDIT, 0, _("Console [%s] from [%s] cmdline %s\n"), user_name, host,
         cmd);
 }
+
+void UaContext::LogAuditEventInfoMsg(const char* audit_msg, const char* item)
```

Comment:
```suggestion
void UaContext::LogAuditEventInfoMsg(const char* fmt, ...)
```

Reply:
Would be nice to have an interface that has a format string as input parameter. 

See void UaContext::InfoMsg(const char* fmt, ...) as example.

Reply:
Suggestion implemented